cmd/boulder-va/main.go

```diff
@@ -83,8 +74,6 @@ func main() {
 	var dnsTimeout time.Duration
 	if c.VA.DNSTimeout != "" {
 		dnsTimeout, err = time.ParseDuration(c.VA.DNSTimeout)
-	} else {
-		dnsTimeout, err = time.ParseDuration(c.Common.DNSTimeout)
 	}
```

Comment:
Given that the deleted code here was an unconditional `else` clause, it seems like the remaining code should just be `dnsTimeout = time.ParseDuration(c.VA.DNSTimeout)`, and the DNSTimeout config field should be decorated with `validate:"required"`.

Also, why isn't DNSTimeout a cmd.ConfigDuration? Pretty sure we could make that change on our end without affecting SRE at all, and it would remove the need to call ParseDuration here, allowing us to use c.VA.DNSTimeout.Duration directly on line 107.

Reply:
I thought about changing it but decided that it wasn't in the scope of the change. If you disagree I'm happy to make the change here.